src/config/auth.rst

```diff
@@ -214,7 +214,7 @@ Authentication Configuration
 
     .. config:option:: secret :: Proxy Auth secret token
 
-        The secret token used for :ref:`api/auth/proxy` method. ::
+        The secret token is used for :ref:`api/auth/proxy` and for :ref:`api/auth/cookie`. ::
```

Comment:
I think about just generic "authentication secret token" or something, but you may figure something else and better (: